### PR TITLE
Flashpoint Release 2.0.2

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Updated the indicator extraction settings to only extract certain type of indicators from the relevant incident fields.
- Updated the "flashpoint-ignite-compromised-credentials-list" command for the "username" response field.
- Updated the mapping for the "Flashpoint Compromised Email" incident field to "username" response field. If not found, then it will use the "email" response field.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Flashpoint/Integrations/Ignite/Ignite.py
Comment (amshamah419) [inline Packs/Flashpoint/Integrations/Ignite/Ignite.py:656]: It might be worth adding in a log line to specify which of the fields was selected for the name. Purely for debugging purposes.

**Review by amshamah419 — COMMENTED** [review]
Hi @crestdatasystems Thanks for the contribution. Just one minor note for debugging, but otherwise the code looks good. I'll approve after we get the review on the classifier and incident type from our security team member @idovandijk

**Comment by amshamah419** [comment]
Hi @crestdatasystems On my end, the PR looks good. When we get feedback regarding the incident type changes, I will approve so we can merge.

File: Packs/Flashpoint/Integrations/Ignite/Ignite.py
Comment (amshamah419) [inline Packs/Flashpoint/Integrations/Ignite/Ignite.py:676]: You can spin this off to use a helper function which might be more performative.

Something like this:
```python
def get_incident_name(hit_source: dict) -> str:
    '''
    Determines the incident name based on available fields in the hit source.
    :param hit_source: The source data from the hit.
    :return: The incident name.
    '''
    for field in ['username', 'email', 'fpid']:
        value = hit_source.get(field)
        if value:
            demisto.debug(f'Setting incident name with {field}: {value}')
            return value
    demisto.debug('Setting incident name with default: Compromised Credential Alert')
    return 'Compromised Credential Alert'


def process_hits(hits: list):
    '''
    Processes the hits and generates incidents with the appropriate severity and details.
    :param hits: The list of hits to process.
    :return: List of incident dictionaries.
    '''
    severity = demisto.params().get('severity', DEFAULT_SEVERITY)
    incidents = []

    for hit in hits:
        hit_source = hit.get('_source', {})
        incident_name = get_incident_name(hit_source)

        incidents.append({
            'name': incident_name,
            'severity': getattr(IncidentSeverity, severity.upper(), None),  # safer access to enum
            'occurred': hit_source.get('breach', {}).get('created_at', {}).get('date-time'),
            'rawJSON': json.dumps(hit)
        })

    return incidents
```

**Review by amshamah419 — APPROVED** [review]
Hi @crestdatasystems the code changes look good to me. Just one minor point about the process hits which may be helpful. It's up to you if you'd like to implement it this way.

File: Packs/Flashpoint/Integrations/Ignite/Ignite.py
Comment (crestdatasystems) [reply Packs/Flashpoint/Integrations/Ignite/Ignite.py]: Hi @amshamah419, thanks for suggesting helper function. We have pushed a commit for this.

**Comment by crestdatasystems** [comment]
> Hi @crestdatasystems the code changes look good to me. Just one minor point about the process hits which may be helpful. It's up to you if you'd like to implement it this way.

Thanks @amshamah419, for reviewing PR.

Please let me know if you need anything else from our side in order to finish it!

**Comment by amshamah419** [comment]
Hi @crestdatasystems we're just waiting on a security review in order to progress. Codewise I think you're in good shape. @idovandijk any ETA?
@samuelFain FYI

**Review by idovandijk — COMMENTED** [review]
@crestdatasystems Apologies for the delay and thank you for your patience.
Changes in classifier and incident type make good sense, I'm happy to see usage of specific extractions on field changes.

@amshamah419 this is good to go from my side, feel free to approve it

**Pull request merged**
